google-cloud-storage/README.md

```diff
@@ -99,19 +99,45 @@ Bucket bucket = storage.create(BucketInfo.of(bucketName));
 
 // Upload a blob to the newly created bucket
 BlobId blobId = BlobId.of(bucketName, "my_blob_name");
-Blob blob = bucket.create(
-    "my_blob_name", "a simple blob".getBytes(UTF_8), "text/plain");
+BlobInfo blobInfo = BlobInfo.newBuilder(blobId).setContentType("text/plain").build();
+Blob blob = storage.create(blobInfo, "a simple blob".getBytes(UTF_8));
 ```
 
+A complete example for creating a blob can be found at
+[CreateBlob.java](./google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/CreateBlob.java).
+
 At this point, you will be able to see your newly created bucket and blob on the Google Developers Console.
 
 #### Retrieving data
 Now that we have content uploaded to the server, we can see how to read data from the server.  Add the following line to your program to get back the blob we uploaded.
 
 ```java
-String blobContent = new String(blob.getContent(), UTF_8);
+BlobId blobId = BlobId.of(bucketName, "my_blob_name");
+byte[] content = storage.readAllBytes(blobId);
+String contentString = new String(content, UTF_8);
 ```
 
+A complete example for accessing blobs can be found at
+[CreateBlob.java](./google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/CreateBlob.java).
+
+### Updating data
+Another thing we may want to do is update a blob. The following snippet shows how to update a Storage blob if it exists. 
+
+``` java
+BlobId blobId = BlobId.of(bucketName, "my_blob_name");
+Blob blob = storage.get(blobId);
+if (blob != null) {
+  byte[] prevContent = blob.getContent();
+  System.out.println(new String(prevContent, UTF_8));
+  WritableByteChannel channel = blob.writer();
+  channel.write(ByteBuffer.wrap("Updated content".getBytes(UTF_8)));
+  channel.close();
+}
+```
+
+The complete source code can be found at
+[UpdateBlob.java](./google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/UpdateBlob.java).
+
```

Comment:
(Commenting here, since I don't know how to comment on an unchanged line).
This is the comment for line 162.
It says "we put together all the code shown above into one program", but the new update example is not added in the snippet. Please update the snippet also (and probably rename it, since since it will be doing an update too).